src/main/java/frc/robot/subsystems/LimelightNavigation.java

```diff
@@ -35,53 +43,125 @@ public class LimelightNavigation extends SubsystemBase {
   RelativeEncoder m_RearRightWheel_Encoder;
 
   MecanumDriveKinematics m_Kinematics;
-  MecanumDriveOdometry m_DriveOdometry;
+  // MecanumDriveOdometry m_DriveOdometry;
+  static MecanumDrivePoseEstimator m_DrivePoseEstimator;
+
+  public static Pigeon2 m_pigeon2;
```

Comment:
this should be a `private` instance variable (not `static`) - see comment on `climberSubsystem`

Reply:
Which line? @drewwhis

Reply:
We only need one Pigeon2 device in the whole project.  Per the prior comment, the pigeon should be moved to the RobotContainer and can be made private there, and not static.  

The m_DrivePoseEstimator currently needs to be static because some methods are called from other areas that don't use a pointer to the LimelightNavigation object.  Probably there's some cleanup we can do with that.  I'll take a look for that.

Reply:
Doneski